lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -564,15 +565,41 @@ def get(self, request, course_key):
                     grade_conditions['effective_grade__lte'] = request.GET.get('assignment_grade_max')
                 if request.GET.get('assignment_grade_min'):
                     grade_conditions['effective_grade__gte'] = request.GET.get('assignment_grade_min')
-                annotations['selected_assignment_grade_in_range'] = Exists(
-                    subqueryset.filter(
```

Comment:
It does look like your changes may've clobbered the filter I added for assignment grades.

Reply:
Ack! Sorry about that. I'll fix that.